### ISSUE TITLE
[Bug] multiple scrolls needed to access "quit button"

### ISSUE THREAD

**Issue by clavelc** [mozilla-mobile/fenix#19978]
https://user-images.githubusercontent.com/25869334/121775102-e780d100-cb85-11eb-8845-9902398c8954.mp4

[comment]: # (Every issue should have the exact bug and steps to reproduce described in it. Please do not file feedback list tickets as it is difficult to parse them and address their individual points)
[comment]: # (Read https://github.com/mozilla-mobile/fenix#i-want-to-file-an-issue for more information)

## Steps to reproduce
Enable the quit button in settings
Open a new page in a tab
Open the three-dot menu
Scroll down fast to access the end

### Expected behavior
The bottom end of the menu is shown

### Actual behavior
The menu is made larger, and a second scroll action is needed to access the end of the menu. 

### Device information

* Device vendor / model and Android version: ? 
Galaxy s7, android 9
* Firefox for Android version: ? (go to Settings -> About Firefox)
Tested with latest normal build and latest nightly 91.0a1, same issue in both.

**Comment by SoftVision-LorandJanos** [comment]
Hi @clavelc  ! Thank you for your report! I was able to reproduce the issue on the latest Nightly (Monday 6/14 @ 5:07 AM).
However it seems to be device dependent. As I had a Samsung Galaxy Tab 3 (Android 9), I've tried it on that one, but one scroll got the bottom view. On a Google Pixel 4  with Android 11 I had the same result. However the device I was able to reproduce it on is a Motorola Nexus 6 with Android 7.1.1.
